app/views/dashboards/_activity_feed.html.erb

```diff
@@ -0,0 +1,6 @@
+<div id='activity_feed' class='margin-bottom'>
+  <h1 class='border-bottom title'>Activity Feed</h1>
+  <% Event.order_by(created_at: :desc).limit(10).each do |event| %>
```

Comment:
Let's move the db call into the controller and also filter on `where(line: current_user.line)`, if that's okay!